### PR TITLE
Fix "the point" problems

### PR DESCRIPTION
**Pull request by xfq**
Point is a proper name when it refers to the current editing location.
Moreover, point designates a place *between* two characters (or before the first
character, or after the last character), rather than a particular character.

References:

* http://lists.gnu.org/archive/html/emacs-devel/2008-10/msg00414.html
* https://www.gnu.org/software/emacs/manual/html_node/elisp/Point.html#index-point

### CODE REVIEW

File: core/core-dotspacemacs.el [hunk 1]
```diff
@@ -222,9 +222,10 @@ can be toggled through `toggle-transparency'.")
   "If non nil unicode symbols are displayed in the mode-line (eg. for lighters)")
 
 (defvar dotspacemacs-smooth-scrolling t
-  "If non nil smooth scrolling (native-scrolling) is enabled. Smooth scrolling
-overrides the default behavior of Emacs which recenters the point when
-it reaches the top or bottom of the screen.")
+  "If non nil smooth scrolling (native-scrolling) is enabled.
+Smooth scrolling overrides the default behavior of Emacs which
+recenters point when it reaches the top or bottom of the
```
Comment (StreakyCobra) [inline core/core-dotspacemacs.el]: You probably need to synchronize `core/templates/.spacemacs.template` to keep both synchronized :-)

File: doc/CONVENTIONS.org [hunk 2]
```diff
@@ -158,7 +158,7 @@ Live evaluation of code is under the prefix ~SPC m e~.
 
 | Key     | Description                                       |
 |---------+---------------------------------------------------|
-| ~m e $~ | put the point at the end of the line and evaluate |
+| ~m e $~ | put point at the end of the line and evaluate |
```
Comment (StreakyCobra) [inline doc/CONVENTIONS.org]: You need to align the table (ending `|`)

**Comment by xfq** [comment]
@StreakyCobra Thanks.  Should be fixed now.

**Comment by StreakyCobra** [comment]
:+1:

**Comment by StreakyCobra** [comment]
Thank you :+1: Cherry-picked into the `develop` branch. You can safely delete your branch :smiley:

I had to rebase it and I squashed the two commits (4958f7fc10c58c39dd30085261b943c2ab1b3b12), may be worth a second look then :-) I also aligned the table in SQL readme (56115e9bce117eee97388fe0ad2f69716528236b)

**Comment by xfq** [comment]
It seems all right.

Thank you @StreakyCobra !

**Pull request closed without merging**